webpack.common.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path')
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+
+module.exports = {
+  entry: {
+    app: './src/index.js'
+  },
+    output: {
+      filename: 'index.bundle.js',
+      path: path.resolve(__dirname, 'public')
+    },
+
+    module: {
+    rules: [
+      {
+        test: /\.(s*)css$/,
+        use: ExtractTextPlugin.extract({
+        	use:[
+        	  { loader: 'css-loader', options: { minimize: true }},
+        	  { loader: 'sass-loader?sourceMap'}
+        	]
+        })
+      },
+      {
+        test: /\.pug/,
+        use: ['pug-loader']
+      },
+      {
+        test: /\s[a|c]ss$/,
+        exclude: /node_modules/,
+        loader: 'sasslint'
+      }
```

Comment:
I would suggest to use the [enforce](https://webpack.js.org/configuration/module/#rule-enforce) config for any kind of linting loader.